packages/apex-node/src/utils/traceFlags.ts

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Connection } from '@salesforce/core';
+import { nls } from '../i18n';
+import {
+  DEFAULT_DEBUG_LEVEL_NAME,
+  LOG_TIMER_LENGTH_MINUTES,
+  LOG_TYPE
+} from '../logs/constants';
```

Comment:
let's export these constants at the top level of the `logs` folder from the `index.ts` file